.gitignore

```diff
@@ -0,0 +1 @@
+/nbproject
```

Comment:
it should got to your local gitignore (I assume that directory is connected to your editor)